deployment/src/strongmind_deployment/rails.py

```diff
@@ -55,7 +58,6 @@ def __init__(self, name, opts=None, **kwargs):
         :key autoscale: Whether to autoscale the web container. Defaults to True.
         :key worker_autoscale: Whether to autoscale the worker container. Defaults to False.
         :key db_engine_version: The version of the database engine. Defaults to 15.4.
-        :key desired_web_count: The number of instances of the web container to run. Defaults to 1.
```

Comment:
we might want to comb through all of our rails projects and see if they are using this. We will likely want to remove usages of this so it's not confusing for people.